Kinvey-Xamarin/Offline/SQLTemplates.cs

```diff
@@ -82,14 +82,6 @@ public OfflineMetaData(string id, JObject customHeaders)
 
 			public OfflineMetaData(){}
```

Comment:
@vinaygahlawat I happened to notice this line in the same file. Is this still relevant?